### PR TITLE
#1994 feat: Unify keyboard notifications logic

### LINKED ISSUE #1994: Unify keyboard notifications logic
Separate keyboard notifications observation logic, so it's easy to use in any view controller

https://github.com/FlowCrypt/flowcrypt-ios/blob/master/FlowCrypt/Controllers/Setup/SetupCreatePassphraseAbstractViewController.swift#L104

### PR DESCRIPTION
**Pull request by ioanmo226**
This PR unified keyboard notifications logic

close #1994 // if this PR closes an issue

----------------------------------

**Tests** _(delete all except exactly one)_:
- Does not need tests (refactor only, docs or internal changes)

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [ ] addresses the issue it closes (if any)
- [ ] code is readable and understandable
- [ ] is accompanied with tests, or tests are not needed
- [ ] is free of vulnerabilities
- [ ] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

File: FlowCryptUI/Nodes/TableViewController.swift [hunk 1]
```diff
@@ -26,6 +26,28 @@ open class TableNodeViewController: ASDKViewController<TableNode> {
...
+            selector: #selector(adjustForKeyboard),
+            name: UIResponder.keyboardWillHideNotification,
+            object: nil
+        )
+    }
+
+    @objc open func adjustForKeyboard(notification: Notification) {
+        let height = self.keyboardHeight(from: notification)
+        let insets = UIEdgeInsets(top: 0, left: 0, bottom: height + 5, right: 0)
+        node.contentInset = insets
```
Comment (sosnovsky) [inline FlowCryptUI/Nodes/TableViewController.swift:49]: It seems we're missing `node.scrollToRow(at: IndexPath(item: Parts.passPhrase.rawValue, section: 0), at: .middle, animated: true)` here

Comment (ioanmo226) [reply FlowCryptUI/Nodes/TableViewController.swift:49]: You are right Nice catch. 👍 
Just fixed

Comment (sosnovsky) [reply FlowCryptUI/Nodes/TableViewController.swift:49]: `SetupManuallyEnterPassPhraseViewController` is still missing this `scrollToRow` addition

Comment (ioanmo226) [reply FlowCryptUI/Nodes/TableViewController.swift:49]: Fixed

**Review by sosnovsky — APPROVED** [review]

**Pull request merged**